src/assets/styles/collections/_projects.css

```diff
@@ -129,25 +129,31 @@
 
 @media (width >= 70.875rem) {
     .project .banner {
-        box-shadow:
-            -75rem 0 0 0 var(--fl-fgColor, var(--project-color)),
-            inset 3.5rem 0 0 0 var(--fl-fgColor, var(--project-color)),
-            inset 0 -0.1875rem 0 0 var(--fl-fgColor, transparent);
-        flex-direction: row;
-        height: 39.625rem;
+        display: grid;
+        gap: 0;
+        grid-template-columns: 35% 65%;
+        grid-template-rows: 1;
```

Comment:
This is causing unexpected behaviour in a project page without banner-image. Instead of using `grid-template-rows` to create a single row, I would recommend using `grid-auto-flow: column`.